Sources/RoktUXHelper/UI/Components/EmbeddedComponent/EmbeddedComponentViewModel.swift

```diff
@@ -56,9 +56,13 @@ class EmbeddedComponentViewModel: ObservableObject {
     }
 
     func updateHeight(_ newHeight: CGFloat) {
-        if lastUpdatedHeight != newHeight {
+        let roundedHeight = round(newHeight)
+        let roundedLastUpdatedHeight = round(lastUpdatedHeight)
```

Comment:
Would suggest forcibly rounding up to prevent any cropping of descenders on lettering, underlines, etc. Not sure on exact code but something like

```suggestion
        let roundedHeight = newHeight.rounded(.up)
        let roundedLastUpdatedHeight = lastUpdatedHeight.rounded(.up)
```

Reply:
Good call, done.